node/src/effect.rs

```diff
@@ -735,6 +735,7 @@ impl<REv> EffectBuilder<REv> {
     }
 
     /// Gets the requested block header from the linear block store.
+    #[allow(unused)]
```

Comment:
Really, we aren't using that anywhere now?
I guess we'll need it again soon, because in some places where we currently get the block, we only need the header?

Reply:
That's what the compiler says.
And yes, I prefer to leave it exactly for the reason you mention - I think there might be places where we'll be switching from getting a block to getting a header.